### PR TITLE
Remove version

### PR DESCRIPTION
**Pull request by Aboudjem**
Removed version from constructor, hardcoded version "3.0"

Updated tests

### CODE REVIEW

File: contracts/token/Token.sol [hunk 1]
```diff
@@ -81,14 +79,13 @@ contract Token is IToken, AgentRole {
...
         uint8 _decimals,
-        string memory _version,
         address _onchainID
         )
     public {
         tokenName = _name;
         tokenSymbol = _symbol;
         tokenDecimals = _decimals;
-        tokenVersion = _version;
+        tokenVersion = "3.0";
```
Comment (Nakasar) [inline contracts/token/Token.sol:88]: Maybe include hotfix version as well?

```suggestion
        tokenVersion = "3.0.0";
```

**Comment by Nakasar** [comment]
I think we can directly declare the variable as constant and remove it from the constructor.

**Comment by Nakasar** [comment]
This is also a breaking change to mirror in our SDK/documentation.

**Comment by Joachim-Lebrun** [comment]
i agree that, in this case it is better to put it as a constant directly and remove it prom the constructor, logically the event associated, `UpdatedTokenInformation(tokenName, tokenSymbol, tokenDecimals, tokenVersion, tokenOnchainID)` should not include the version in this case but it would break the tracking done on thegraph

**Comment by Aboudjem** [comment]
> i agree that, in this case it is better to put it as a constant directly and remove it from the constructor, logically the event associated, `UpdatedTokenInformation(tokenName, tokenSymbol, tokenDecimals, tokenVersion, tokenOnchainID)` should not include the version in this case but it would break the tracking done on thegraph

Yes that's why I let the UpdatedTokenInformation, I think it's better to let it like that so we don't have to do changes on Graph and everytime a Token is deployed we have all the information at once

**Comment by Myna65** [comment]
> i agree that, in this case it is better to put it as a constant directly and remove it from the constructor, logically the event associated, `UpdatedTokenInformation(tokenName, tokenSymbol, tokenDecimals, tokenVersion, tokenOnchainID)` should not include the version in this case but it would break the tracking done on thegraph

Or we can keep it in the event and just hardcode it there too

**Comment by Nakasar** [comment]
Increase package-lock version to "3.0.0-betaX"

File: contracts/token/Token.sol [hunk 2]
```diff
@@ -69,8 +69,6 @@ contract Token is IToken, AgentRole {
     *  @param _name the name of the token
     *  @param _symbol the symbol of the token
     *  @param _decimals the decimals of the token
-    *  @param _version the version of the token, current version is 3.0
-    *  @param _onchainID the address of the onchainID of the token
```
Comment (Joachim-Lebrun) [inline contracts/token/Token.sol:73]: why did you remove this?

Comment (Joachim-Lebrun) [reply contracts/token/Token.sol:73]: _onchainID is still a parameter

Comment (Aboudjem) [reply contracts/token/Token.sol:73]: > _onchainID is still a parameter

Didn't notice that I have deleted this, fixed :)
